db/schemas/manifestSchema20160105.json

```diff
@@ -0,0 +1,367 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id":"https://e-manifest.18f.gov/schemas/manifestSchema20160105.json",
+    "title": "e-Manifest JSON Schema",
+    "description": "First draft of JSON Schema.  I fully expect this to be editted and revised.  Resource used to make this was http://spacetelescope.github.io/understanding-json-schema/index.html",
+
+    "definitions":{
+        "address": {
+             "type": "object",
+            "properties":{
+                "address_1":   {
+                    "description": "Address line 1",
+                    "type": "string",
+                    "maxLength":30 ,
+                    "minLength":0 
```

Comment:
Should these actually be something like `"minLength": 1` so that we get a validation error if these values are not supplied?